src/test/java/org/fcrepo/camel/integration/FcrepoBinaryHeadIT.java

```diff
@@ -25,38 +25,40 @@
 import org.apache.camel.Produce;
 import org.apache.camel.ProducerTemplate;
 import org.apache.camel.builder.RouteBuilder;
-import org.apache.camel.builder.xml.Namespaces;
 import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.support.builder.Namespaces;
 import org.apache.camel.test.junit4.CamelTestSupport;
 import org.apache.jena.vocabulary.RDF;
 import org.fcrepo.camel.FcrepoHeaders;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Test adding a non-RDF resource
  * @author Aaron Coburn
  * @since November 7, 2014
  */
+@Ignore
```

Comment:
This test seemed to pass for me when I removed the `@Ignore`, maybe you can reenable it?